src/remote.sh

```diff
@@ -52,13 +54,19 @@ function cmd_remotely()
     composed_cmd="ssh -F ${configurations['ssh_configfile']} ${configurations['hostname']}"
   fi
 
+  iterative_output=${iterative_output:-''}
```

Comment:
I think this line adds nothing, right? If "$7" is not supplied, then `iterative_output` has the same value as the default, which is `''` (null string).